content/4.examples/send-payment.md

```diff
@@ -72,14 +74,14 @@ sequenceDiagram
 
 ::notice{info}
 
-Check more info about payment query [here](query-payment-states)
+Find more information about payment queries [here](query-payment-states).
```

Comment:
This kind of link can work in different way depending on trailing slash at your current location.

```suggestion
Find more information about payment queries [here](/examples/query-payment-states).
```